### PR TITLE
DOCS-806: Update GPS RTK model docs

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Configure an NTRIP-based RTK GPS with an I2C Connection"
+linkTitle: "gps-nmea-rtk-pmtk"
+weight: 10
+type: "docs"
+description: "Configure an NTRIP-based RTK that uses I2C communication."
+images: ["/icons/components/imu.svg"]
+# SMEs: Susmita
```
Comment (npentrel) [inline docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md:8]: make sure to add an alias if this is replacing gps-rtk.

**Review by susmitaSanyal — CHANGES_REQUESTED** [review]
I wonder if it would be useful to give instructions on how to get NTRIP server url, user and pass.

File: docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md (hunk 1 above)
Comment (JessamyT) [reply docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md:8]: I already added the alias on the serial one :D Dunno if I should switch it to this one instead? Figured it was either one.

File: docs/components/movement-sensor/gps/gps-nmea-rtk-serial.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-sensor-name>",
+      "type": "movement_sensor",
+      "model": "gps-nmea-rtk-serial",
+      "attributes": {
+        "board": "<your-board-name>",
```
[the remark this answers is not in the document]
Comment (JessamyT) [reply docs/components/movement-sensor/gps/gps-nmea-rtk-serial.md:61]:
```suggestion
```

File: docs/components/movement-sensor/gps/gps-nmea-rtk-serial.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "my_GPS",
+      "type": "movement_sensor",
+      "model": "gps-nmea-rtk-serial",
+      "attributes": {
+        "board": "local",
+        "serial_path": "/dev/ttyS0",
```
[the remark this answers is not in the document]
Comment (JessamyT) [reply docs/components/movement-sensor/gps/gps-nmea-rtk-serial.md:88]:
```suggestion
        "serial_path": "/dev/serial/by-path/ttyS0",
```

**Comment by JessamyT** [comment]
@susmitaSanyal Is there a doc or other source you can point me to that has those instructions?

**Comment by susmitaSanyal** [comment]
@JessamyT do you mean Ntrip setup instruction?

**Comment by JessamyT** [comment]
@susmitaSanyal yeah

**Comment by susmitaSanyal** [comment]
@JessamyT There's isn't any, but I can put together one.

**Comment by JessamyT** [comment]
Okay! That'd be great, thanks so much! How about I make a docs ticket to add that later and we merge this for now?

File: docs/components/movement-sensor/gps/rtk-station.md [hunk 4]
```diff
@@ -133,6 +133,8 @@ Edit and fill in the attributes as applicable.
 }
 ```
 
+Find attribute details in the [GPS-RTK documentation](../gps-nmea-rtk-pmtk/).
+
```
Comment (susmitaSanyal) [inline docs/components/movement-sensor/gps/rtk-station.md:137]: You can remove rtk-station from GPS

**Review by susmitaSanyal — CHANGES_REQUESTED** [review]
Let's remove the rtk-station.

File: docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md (hunk 1 above)
Comment (npentrel) [reply docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md:8]: nvm I missed that

Comment (npentrel) [reply docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md:8]: actually - what about rtk-station?

Comment (susmitaSanyal) [reply docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md:8]: @npentrel rtk-station has been removed from rdk. It's not a component.

Comment (susmitaSanyal) [reply docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md:8]: @npentrel rtk-station has been removed from RDK, it's not a component anymore.

**Review by susmitaSanyal — APPROVED** [review]
LGTM!

**Pull request merged**